apps/web/src/ee/clerk/components/QuestionnaireForm.tsx

```diff
@@ -45,19 +47,16 @@ export function QuestionnaireForm() {
   const location = useLocation();
   const [_, setParams] = useSearchParams();
 
-  const { mutateAsync: updateOrganizationMutation } = useMutation<{ _id: string }, IResponseError, any>(
-    (data: UpdateExternalOrganizationDto) => updateClerkOrgMetadata(data)
-  );
-
   async function updateOrganization(data: UpdateExternalOrganizationDto) {
     const selectedLanguages = Object.keys(data.language || {}).filter((key) => data.language && data.language[key]);
 
     const updateClerkOrgDto: UpdateExternalOrganizationDto = {
       jobTitle: data.jobTitle,
       domain: data.domain,
       language: selectedLanguages,
+      companySize: data.companySize,
     };
-    await updateOrganizationMutation(updateClerkOrgDto);
+    await updateClerkOrgMetadata(updateClerkOrgDto);
```

Comment:
We need to also update these 2 things:
- backend endpoint logic and DTOs, ref: https://github.com/novuhq/packages-enterprise/blob/next/auth/src/controllers/clerk.metadata.controller.ts#L27
- Clerk public metadata type so that its typed, ref: https://github.com/novuhq/novu/blob/next/packages/shared/src/types/clerk/types.ts
  - btw here you can also remove `defaultLocale` and `apiServiceLevel` from the type since we dont store that in Clerk